redash/__init__.py

```diff
@@ -45,5 +47,9 @@ def setup_logging():
 statsd_client = StatsClient(host=settings.STATSD_HOST, port=settings.STATSD_PORT, prefix=settings.STATSD_PREFIX)
 limiter = Limiter(key_func=get_ipaddr, storage_uri=settings.LIMITER_STORAGE)
 
+def enqueue(queue, func, *args, **kwargs):
```

Comment:
Can't we move this into `redash.queues` or something? Putting more into `redash/__init__.py` is kind of backwards in terms of maintainability regrading possible import-time-side-effects.

Reply:
Hehe, I'm guessing this is the downside of a draft PR - I created the `enqueue` function temporarily to reduce duplication and pushed. Planned on finding an appropriate place later :)